test/regex.spec.js

```diff
@@ -0,0 +1,87 @@
+import assert from "assert";
+
+describe("RegEx", function () {
```

Comment:
Not sure how you would want to document this, in the code, Readme, or otherwise, but just thought I'd share that my favorite RegEx tool is https://regexr.com/

Reply:
Added to readme